### PR TITLE
YPJT and Perth VFR fix

### PR DESCRIPTION
**Pull request by Verbjimtim**
Fix to [issue #131](https://github.com/vatSys/australia-dataset/issues/131) and more. 

YPJT position 
image

Perth Position with VFR
image
now differentiating the Victor routes as they are INSIDE controlled airspace.

### CODE REVIEW

File: Maps/Perth/PH_TCU.xml [hunk 1]
```diff
@@ -334,10 +314,108 @@
...
+    </Label>
+    <Symbol Type="HollowSquare">
+      <Point>SHOP</Point>
+    </Symbol>
+    <!--Fremantle Golfcourse Departure-->
+    <Line Pattern="Hash">
+    -32.07036705892901+115.84903482859521/FREM
+    </Line>
+    <Label LabelOrientation="SE">
+      <Point Name="FREEMANTLE DEPARTURE">-32.07036705892901+115.84903482859521</Point>
```
Comment (alphadelta332) [inline Maps/Perth/PH_TCU.xml:396]: Could you please fix the spelling of **Fremantle** :)

File: Maps/Perth/PH_TCU.xml [hunk 2]
```diff
@@ -334,10 +314,108 @@
...
+      <Point Name="POWER HOUSE">POWR</Point>
+      <Point Name="ADVENTURE WORLD">ADWD</Point>
+      <Point Name="BOATYARD">BOAT</Point>
+      <Point Name="YANGEBUP LAKE">YGB</Point>
+      <Point Name="THOMSONS LAKE">TOML</Point>
+      <Point Name="RUSSELL">RUSS</Point>
+      <Point Name="OAKFORD">OAKF</Point>
+      <Point Name="FORRESTDALE LAKE">FDL</Point>
+      <Point Name="ARMADALE SHOPS 'SHOPS'">SHOP</Point>
+      <Point Name="FREEMANTLE GOLFCOURSE">FREM</Point>
```
Comment (alphadelta332) [inline Maps/Perth/PH_TCU.xml:416]: **Fremantle** spelling as above

File: Maps/Perth/PH_TCU.xml [hunk 3]
```diff
@@ -334,10 +314,108 @@
...
+  <Map Type="System2" Name="YPJT_FULL NAMES" Priority="1" Center="-32.098611+115.8808">
+    <Label HasLeader="True" LabelOrientation="SW">
+      <Point Name="POWER HOUSE">POWR</Point>
+      <Point Name="ADVENTURE WORLD">ADWD</Point>
+      <Point Name="BOATYARD">BOAT</Point>
+      <Point Name="YANGEBUP LAKE">YGB</Point>
+      <Point Name="THOMSONS LAKE">TOML</Point>
+      <Point Name="RUSSELL">RUSS</Point>
+      <Point Name="OAKFORD">OAKF</Point>
+      <Point Name="FORRESTDALE LAKE">FDL</Point>
```
Comment (alphadelta332) [inline Maps/Perth/PH_TCU.xml:414]: **Forestdale** Lake spelling :)

**Review by alphadelta332 — CHANGES_REQUESTED** [review]

File: Maps/Perth/PH_TCU.xml (hunk 1 above)
Comment (Verbjimtim) [reply Maps/Perth/PH_TCU.xml:396]: Will push to fix

File: Maps/Perth/PH_TCU.xml (hunk 2 above)
Comment (Verbjimtim) [reply Maps/Perth/PH_TCU.xml:416]: both XD

File: Maps/Perth/PH_TCU.xml (hunk 3 above)
Comment (Verbjimtim) [reply Maps/Perth/PH_TCU.xml:414]: This one is correct, they be pirates arrr 
image

**Review by alphadelta332 — CHANGES_REQUESTED** [review]

File: Maps/Perth/PH_TCU.xml [hunk 4]
```diff
@@ -314,17 +295,16 @@
...
       <Point Name="OBSERVATION CITY">OBC</Point>
       <Point Name="PERTH CITY">PCTY</Point>
       <Point Name="PICKERING BROOK GOLF COURSE">PIB</Point>
-      <Point Name="POWERHOUSE">POWR</Point>
       <Point Name="REDCLIFFE BRIDGE">REDC</Point>
       <Point Name="ROCKINGHAM">ROHM</Point>
       <Point Name="ROLEYSTONE">RLY</Point>
       <Point Name="RUSSELL">RUSS</Point>
       <Point Name="SAWYERS VALLEY">SWY</Point>
+      <Point Name="ARMIDALE 'SHOPS'">SHOP</Point>
```
[the remark this answers is not in the document]
Comment (Verbjimtim) [reply Maps/Perth/PH_TCU.xml:307]: lol I should be more careful next time. I'm happy to remove the inverted commas but most local refer to it as "SHOPS" so wanted to signify that some how.

**Review by alphadelta332 — APPROVED** [review]

**Comment by zkgell** [comment]
image
Hi all, can you please confirm this issue still exists. Above is an image from 2409b and seems those waypoints are there

**Comment by Verbjimtim** [comment]
Yes. 

changes made in file fixes missing inbound line from BOAT to ADWD, separates YPJT routes from Victor routes (with differing lines to signify class C entry), and makes a visible difference between YPJT inbounds and outbounds.

New PH VFR map (no YPJT routes as they aren't required for YPPH APP when not top downing YPJT)
image
New YPJT VFR map a dedicated map with accurate tracking points
image

**Comment by zkgell** [comment]
@Slep-wt thoughts? Looks like it diverges from standards across the dataset. Either all need to be replaced or none.

**Comment by Slep-wt** [comment]
> @Slep-wt thoughts? Looks like it diverges from standards across the dataset. Either all need to be replaced or none.

Agreed, @Verbjimtim can you please do this for the rest of the Metro-D aerodromes please?

**Comment by Verbjimtim** [comment]
@Slep-wt @zkgell All the Metro Ds completed on the new styling. the only one that is lacking is Moorabbin and Bankstown as majority their departures are circuit leg departures. which would be unrealistic to draw XD happy for notes, LMK

YPJT
Screenshot 2024-09-30 191850

YBAF
Screenshot 2024-10-06 171750

YPPF
Screenshot 2024-10-07 210725

YSBK
Screenshot 2024-10-07 143328

YMMB
Screenshot 2024-10-09 164250

Not for future self YMMB's SHOAL renames to SHOL next AIRAC

**Comment by Slep-wt** [comment]
> @Slep-wt @zkgell All the Metro Ds completed on the new styling. the only one that is lacking is Moorabbin and Bankstown as majority their departures are circuit leg departures. which would be unrealistic to draw XD happy for notes, LMK
> 
> 
> 
> YPJT
> 
> Screenshot 2024-09-30 191850
> 
> 
> 
> YBAF
> 
> Screenshot 2024-10-06 171750
> 
> 
> 
> YPPF
> 
> Screenshot 2024-10-07 210725
> 
> 
> 
> YSBK
> 
> Screenshot 2024-10-07 143328
> 
> 
> 
> YMMB
> 
> image
> 
> 
> 
> Note for future self YMMB's SHOAL renames to SHOL next AIRAC

Sounds good @Verbjimtim! If possible could you rebase the pull request to the 2411draft branch please?

**Comment by Verbjimtim** [comment]
Happy to @Slep-wt how ever I can only see up to 2409draft....

**Comment by Verbjimtim** [comment]
READY! :D

**Pull request merged**